packages/in_app_purchase/in_app_purchase_ios/lib/src/in_app_purchase_ios_platform.dart

```diff
@@ -0,0 +1,223 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/services.dart';
+import 'package:in_app_purchase_ios/src/in_app_purchase_ios_platform_addition.dart';
+import 'package:in_app_purchase_platform_interface/in_app_purchase_platform_interface.dart';
+
+import '../in_app_purchase_ios.dart';
+import '../store_kit_wrappers.dart';
+
+/// [IAPError.code] code for failed purchases.
+final String kPurchaseErrorCode = 'purchase_error';
```

Comment:
Sorry old habit,  in Java and C# (and probably other languages as well) having a public const is considered a code smell. Reason is that compile time the const is replaced with the literal value in consuming libraries. This could lead to bugs when you update the const value and replace the library without re-compiling all dependent code. 

I guess this is not a problem in Dart since code is always compiled entirely including dependencies. I have updated it to `const`.